WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/EpilogueBuilder.kt

```diff
@@ -0,0 +1,87 @@
+package org.wordpress.android.ui.bloggingreminders
+
+import android.os.Build.VERSION_CODES
+import androidx.annotation.RequiresApi
+import org.wordpress.android.R.drawable
+import org.wordpress.android.R.string
+import org.wordpress.android.fluxc.model.BloggingRemindersModel
+import org.wordpress.android.ui.bloggingreminders.BloggingRemindersItem.Caption
+import org.wordpress.android.ui.bloggingreminders.BloggingRemindersItem.HighEmphasisText
+import org.wordpress.android.ui.bloggingreminders.BloggingRemindersItem.Illustration
+import org.wordpress.android.ui.bloggingreminders.BloggingRemindersItem.Title
+import org.wordpress.android.ui.bloggingreminders.BloggingRemindersViewModel.UiState.PrimaryButton
+import org.wordpress.android.ui.utils.ListItemInteraction
+import org.wordpress.android.ui.utils.UiString.UiStringRes
+import org.wordpress.android.ui.utils.UiString.UiStringResWithParams
+import org.wordpress.android.ui.utils.UiString.UiStringText
+import java.time.DayOfWeek
+import java.time.format.TextStyle
+import java.util.Locale
+import javax.inject.Inject
+
+class EpilogueBuilder @Inject constructor(
+    private val dayLabelUtils: DayLabelUtils
+) {
+    @RequiresApi(VERSION_CODES.O) // TODO: Remove this annotation once de-sugaring PR is merged
+    fun buildUiItems(
+        bloggingRemindersModel: BloggingRemindersModel?
+    ): List<BloggingRemindersItem> {
+        val numberOfTimes = dayLabelUtils.buildNTimesLabel(bloggingRemindersModel)
+        val enabledDays = bloggingRemindersModel?.enabledDays?.map { DayOfWeek.valueOf(it.name) }
```

Comment:
Since `enabledDays` is a set and a set doesn't guarantee the order of its elements, we should probably call `.sorted()` after mapping to `DayOfWeek` here, just to be safe.

Reply:
I've refactored formatting as shown in the first screen shot above.  It is the closest to the spec that I could get.